audioipc/src/rpccore.rs

```diff
@@ -220,12 +248,11 @@ impl<C: Client> Handler for ClientHandler<C> {
         trace!("ClientHandler::consume");
         // `proxy` identifies the waiting Proxy expecting `response`.
         if let Some(proxy) = self.in_flight.pop_front() {
-            self.proxies.deliver(proxy, response);
+            if let Err(e) = self.proxies.deliver(proxy, response) {
```

Comment:
Is a silent error enough? Does it need to return an `Err`?

Reply:
Changed to let the error bubble up to the caller.